bigquery_etl/query_scheduling/task.py

```diff
@@ -185,6 +186,8 @@ class Task:
     # manually specified upstream dependencies
     depends_on: List[TaskRef] = attr.ib([])
     depends_on_fivetran: List[FivetranTask] = attr.ib([])
+    # task trigger rule, used to override default of "all_success"
+    trigger_rule: Optional[str] = attr.ib(None)
```

Comment:
Could this be added to the docs here: https://github.com/mozilla/bigquery-etl/blob/main/docs/reference/scheduling.md